upb/def.c

```diff
@@ -2533,18 +2389,11 @@ static bool symtab_add(upb_symtab *s, upb_def *const*defs, size_t n,
 oom_err:
```

Comment:
line 2379 above appears unnecessary now since the symtab should not contain anything with the same name